### PR TITLE
Remove a number of unnecessary key words and modifiers

### PR DESCRIPTION
**Pull request by DanVanAtta**

### CODE REVIEW

File: src/test/java/games/strategy/engine/framework/startup/mc/GameSelectorModelTest.java [hunk 1]
```diff
@@ -101,7 +101,7 @@ private void testObjectSetMockGameData() {
   }
 
   private void verifyTestObjectObserverUpdateSent() {
-    verify(mockObserver, times(1)).update((Observable) Mockito.any(), Mockito.any());
```
Comment (ssoloff) [inline src/test/java/games/strategy/engine/framework/startup/mc/GameSelectorModelTest.java]: `Observable` is now an unused import.

File: src/test/java/games/strategy/net/MessengerTest.java [hunk 2]
```diff
@@ -269,7 +269,7 @@ public void messengerInvalid(final IMessenger messenger, final Exception reason)
   }
 
   @Test
-  public void testManyClients() throws UnknownHostException, CouldNotLogInException, IOException {
```
Comment (ssoloff) [inline src/test/java/games/strategy/net/MessengerTest.java]: `UnknownHostException` is now an unused import.

Comment (DanVanAtta) [inline src/test/java/games/strategy/net/MessengerTest.java]: done - 2f5c4ff

File: src/test/java/games/strategy/engine/framework/startup/mc/GameSelectorModelTest.java (hunk 1 above)
Comment (DanVanAtta) [inline src/test/java/games/strategy/engine/framework/startup/mc/GameSelectorModelTest.java]: done - 2f5c4ff

**Comment by DanVanAtta** [comment]
@ssoloff  updated

**Pull request merged**